jobs/ingestor_cloudwatch/spec

```diff
@@ -0,0 +1,204 @@
+    description: The maximum number of acked events before forcing a checkpoint.
+    default: 1024
+  logstash.queue.checkpoint.writes:
+    description: The maximum number of written events before forcing a checkpoint.
+    default: 1024
+  logstash.queue.checkpoint.interval:
+    description: The interval in milliseconds when a checkpoint is forced on the head page.
+    default: 1000
+
+  logstash_ingestor.filters:
+    description: Filters to execute on the ingestors
+    default: ""
+
+  logstash_ingestor.cloudwatch.port:
+    description: Port to listen for cloudwatch messages
+  logstash_ingestor.cloudwatch.transport:
+    description: Transport protocol to use
+    default: "tcp"
+  logstash_ingestor.cloudwatch.use_keepalive:
+    description: Instruct the socket to use TCP keep alives
+
+  logstash_ingestor.health.disable_post_start:
+    description: Skip post-start health checks?
+    default: false
+  logstash_ingestor.health.interval:
+    description: Logstash cloudwatch health check interval (seconds)
+    default: 5
+  logstash_ingestor.health.timeout:
+    description: Logstash cloudwatch health check number of attempts (seconds)
+    default: 300
+
+  logstash_ingestor.cloudwatch_tls.port:
+    description: Port to listen for cloudwatch-TLS messages (omit to disable)
+  logstash_ingestor.cloudwatch_tls.ssl_cert:
+    description: cloudwatch-TLS SSL certificate (file contents, not a path) - required if logstash_ingestor.cloudwatch_tls.port set
+  logstash_ingestor.cloudwatch_tls.ssl_key:
+    description: cloudwatch-TLS SSL key (file contents, not a path) - required if logstash_ingestor.cloudwatch_tls.port set
+  logstash_ingestor.cloudwatch_tls.skip_ssl_validation:
+    description: Verify the identity of the other end of the SSL connection against the CA.
+    default: false
+  logstash_ingestor.cloudwatch_tls.use_keepalive:
+    description: Instruct the socket to use TCP keep alives
+  logstash_ingestor.cloudwatch.prefix:
+    description: "What prefix in cloudwatch you want to pull in"
+    default: ["/aws/rds/instance/"]
+
+  logstash_ingestor.relp.port:
+    description: Port to listen for RELP messages
+    default: 2514
+
+  logstash_parser.debug:
+    description: Debug level logging
+    default: false
+  logstash_parser.message_max_size:
+    description: "Maximum log message length.  Anything larger is truncated (TODO: move this to ingestor?)"
+    default: 1048576
+  logstash_parser.filters:
+    description: "The configuration to embed into the logstash filters section. Can either be a set of parsing rules as a string or a list of hashes in the form of [{name: path_to_parsing_rules.conf}]"
+    default: ''
+  logstash_parser.deployment_dictionary:
+    description: "A list of files concatenated into one deployment dictionary file. Each file contains a hash of job name-deployment name keypairs for @source.deployment lookup."
+    default: [ "/var/vcap/packages/base-logstash-filters/deployment_lookup.yml" ]
```

Comment:
We're not using these, so probably we should delete them

Reply:
Should be reflected now